src/actions/mainActions.js

```diff
@@ -0,0 +1,40 @@
+import * as types from './actionTypes';
+
+export function setTitleActive() {
+    return {
+        type: types.SET_TITLE_ACTIVE
+    }
+}
+export function setGenreActive() {
+    return {
+        type: types.SET_GENRE_ACTIVE
+    }
+}
+export function setSearchValue(val) {
+    return {
+        type: types.SET_SEARCH_STR,
+        value: val
+    }
+}
+export function sortByReleaseDate() {
+    return {
+        type: types.SORT_RELEASE_DATE
+    }
+}
+export function sortByRating() {
+    return {
+        type: types.SORT_RATING
+    }
+}
+export function fetchMoviesSuccess(movies) {
+    return {
+        type: types.FETCH_MOVIES_SUCCESS,
+        movies
+    }
+}
+export function fetchMovies(url) {

```

Comment:
Probably you can move this action to `movieActions` because it makes more sense to be there, also can be abstracted to another function that can be reused by your other action that fetch movie data.

Reply:
Okay